src/App.js

```diff
@@ -1,32 +1,31 @@
 import React, { useEffect } from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import "./App.scss";
```

Comment:
I think for this one you would probably need to pull my branch again to see what are the final changes.